app/src/main/res/layout/layout1.xml

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Please consider using ctrl + alt  + l for formatting the code. Other than that it looks ok. Good job 👍